### ISSUE TITLE
getUnqiueId() issue in version 10.0.0

### ISSUE THREAD

**Issue by Jade00620** [react-native-device-info/react-native-device-info#1434]
## Unexpected object for getUnqiueId() in version 10.0.0

|             |     |
| ----------- | --- |
| Version     | 10.0.0  |
| Affected OS | ?   |
| OS Version  | ?   |

## Current behavior

Post uprade the version to latest 10.0.0, the value of getUnquieId() is changing to an object instead of a string
<img width="358" alt="image" src="https://user-images.githubusercontent.com/108931673/177976017-c7bf1263-4ff2-4671-9ea4-72278ef37e43.png">

Affect both ios and android

## Expected behavior

Still have the string as output of getUnqiueId()

**Comment by stawecki** [comment]
Hi @Jade00620, 10.0.0 is a major release because it contains this breaking change. See the release notes [here](https://github.com/react-native-device-info/react-native-device-info/releases/tag/v10.0.0) for more info. This behaviour is now matches other sync/async methods in this package.

If you'd like to continue using this method synchronously, try `getUniqueIdSync()`.

**Comment by mikehardy** [comment]
Yep - this was mentioned directly in the release note, it's returning a promise now.
